Forwarded: not-needed
From 24e0628062d6ce6b08678778412877c21244acd6 Mon Sep 17 00:00:00 2001
From: "Mario Limonciello (AMD)" <superm1@kernel.org>
Date: Mon, 10 Nov 2025 22:56:33 -0600
Subject: [PATCH] trivial: fix a test failure on 32 bit x86
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Debian's autopkgtest keeps failing:

```
 91s  ● Showing security
 91s  ● cmd: fwupdtool -v --plugins test security
...
 91s  ● exit code was 0 and expected 1
 91s FAIL: fwupd/fwupdtool.test (Child process exited with code 1)
```

This is because `uname -m` on a 32 bit x86 machine is not `x86`, it's `i686`.

Signed-off-by: Mario Limonciello (AMD) <superm1@kernel.org>
---
 data/tests/fwupdtool.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/tests/fwupdtool.sh b/data/tests/fwupdtool.sh
index 76e0aee88..e27d49b8d 100755
--- a/data/tests/fwupdtool.sh
+++ b/data/tests/fwupdtool.sh
@@ -59,7 +59,7 @@ run export-hwids ${TMPDIR}/hwids.ini
 expect_rc 0
 
 UNAME=$(uname -m)
-if [ "${UNAME}" = "x86_64" ] || [ "${UNAME}" = "x86" ]; then
+if [ "${UNAME}" = "x86_64" ] || [ "${UNAME}" = "i686" ]; then
     EXPECTED=0
 else
     EXPECTED=1
-- 
2.43.0

